pandas/tests/frame/test_constructors.py

```diff
@@ -2499,13 +2497,10 @@ def test_constructor_list_str_na(self, string_dtype):
     def test_dict_nocopy(
         self, request, copy, any_numeric_ea_dtype, any_numpy_dtype, using_array_manager
     ):
-        if using_array_manager and not (
-            (any_numpy_dtype in (tm.STRING_DTYPES + tm.BYTES_DTYPES))
-            or (
-                any_numpy_dtype
-                in (tm.DATETIME64_DTYPES + tm.TIMEDELTA64_DTYPES + tm.BOOL_DTYPES)
-                and copy
-            )
+        if (
+            using_array_manager
+            and not copy
+            and not (any_numpy_dtype in (tm.STRING_DTYPES + tm.BYTES_DTYPES))
```

Comment:
So is it correct to conclude from this remaining skip that this PR fixed the `copy=True` case but not yet the `copy=False` case?

Reply:
correct